### PR TITLE
Work Allocation enhancement - hide show filter button for My Access tab in My work

### PR DESCRIPTION
**Pull request by udaydenduluri33**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/EUI-5087

### Change description ###

1. Work Allocation enhancement - hide show filter button for My Access tab in My work
2. Merge the branch https://github.com/hmcts/rpx-xui-webapp/tree/feature/release3-wa3 into Bookings UI branch

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ X ] No
```

### CODE REVIEW

**Review by tarikdogan-hmcts — APPROVED** [review]

File: api/test/pact/pact-tests/rdJudicial/getJudicialUsers.spec.ts [hunk 1]
```diff
@@ -11,7 +11,7 @@ import { requireReloaded } from '../utils/moduleUtil';
 
 const { Matchers } = require('@pact-foundation/pact');
 import { DateTimeMatcher2 } from '../utils/matchers';
-const { somethingLike, iso8601DateTime, term } = Matchers;
+const { somethingLike,eachLike, iso8601DateTime, term } = Matchers;
```
Comment (connorpgpmcelroy) [inline api/test/pact/pact-tests/rdJudicial/getJudicialUsers.spec.ts:14]: Minor comment - Spacing is off here

File: src/role-access/containers/allocate-role/allocate-role-check-answers/allocate-role-check-answers.component.ts [hunk 2]
```diff
@@ -1,5 +1,6 @@
 import { Component, Input, OnDestroy, OnInit } from '@angular/core';
 import { select, Store } from '@ngrx/store';
+import { RoleCategory } from 'api/roleAccess/models/allocate-role.enum';
```
Comment (connorpgpmcelroy) [inline src/role-access/containers/allocate-role/allocate-role-check-answers/allocate-role-check-answers.component.ts:3]: Think this is an incorrect import

**Review by connorpgpmcelroy — APPROVED** [review]
Approved with couple of minor comments

File: api/test/pact/pact-tests/rdJudicial/getJudicialUsers.spec.ts (hunk 1 above)
Comment (udaydenduluri33) [reply api/test/pact/pact-tests/rdJudicial/getJudicialUsers.spec.ts]: Done

**Pull request merged**